### DISCUSSION TITLE
New Recaptcha on docker

### DISCUSSION THREAD

**Discussion by mhmuc1** [Second-Hand-Friends/kleinanzeigen-bot#232] [Q&A]
Hi,

as the windows-version wasn't working anymore due to the SSL-problem I wanted to try the docker-version.
However when logging in I get the new sliding-captcha. 
Is there a way around that?
Like would it be possible to login on chrome windows and copy the chrome profile with the login-cookie to the data directory of the docker container? If yes: How? ;)

Thank you very much.

**Comment by IMSMarian** [comment]
Hi mhmuc1,
Have you found any solution for this?
I'm wondering the same.
Thank you!

**Comment by mhmuc1** [comment]
Hi, I don't recall how I managed it. However I have a login.bat file that has this content:

```
DEL .\data\userdata\Singlet*

docker run --rm -it --shm-size=256m -e DISPLAY=192.168.1.10:0.0 -v C:\Docker\KleinanzeigenBot\data\:/mnt/data   --entrypoint /usr/bin/chromium   ghcr.io/second-hand-friends/kleinanzeigen-bot   --user-data-dir=/mnt/data/userdata/   --profile-directory=profile   --disable-crash-reporter   --no-first-run   --no-service-autorun   --no-sandbox
```
and publish.bat
`DEL .\data\userdata\Singlet*
docker run -it --shm-size=256m -e DISPLAY=192.168.1.10:0.0 -v C:\Docker\KleinanzeigenBot\data\:/mnt/data  ghcr.io/second-hand-friends/kleinanzeigen-bot  publish`
I hope this helps.
